clients/interfaces/event.go

```diff
@@ -17,7 +17,7 @@ import (
 // EventClient defines the interface for interactions with the Event endpoint on the EdgeX Foundry core-data service.
 type EventClient interface {
 	// Add adds new event.
-	Add(ctx context.Context, req requests.AddEventRequest) (common.BaseWithIdResponse, errors.EdgeX)
+	Add(ctx context.Context, serviceName string, req requests.AddEventRequest) (common.BaseWithIdResponse, errors.EdgeX)
```

Comment:
@hahattan , @cloudxxx8 , The mock for this interface wasn't updated for this. Please submit a PR for the mock.